app/assets/v2/js/grants/index.js

```diff
@@ -25,13 +29,13 @@ $(document).ready(() => {
 
   $('.select2-selection__rendered').removeAttr('title');
 
-  $(document).on('click keypress', '.flip-card', e => {
-    if ($(e.target).is('a') || $(e.target).is('img')) {
-      e.stopPropagation();
-      return;
-    }
-    $(e.currentTarget).toggleClass('turn');
-  });
+  // $(document).on('click keypress', '.flip-card', e => {
+  //   if ($(e.target).is('a') || $(e.target).is('img')) {
+  //     e.stopPropagation();
+  //     return;
+  //   }
+  //   $(e.currentTarget).toggleClass('turn');
+  // });
```

Comment:
Could we remove card_back html + associated CSS as opposed to simple commenting out the JS